src/components/atoms/AdminHeader/AdminHeader.tsx

```diff
@@ -1,4 +1,5 @@
 import React, { Fragment } from "react";
```

Comment:
[Q]
If this component is free of `.module.scss`, has added tailwind classes, why isn't it moved into `./src/components/admin`?

 Is it connected to the inclusion of `@heroicons` (which should probably be replaced by Font Awesome ones)?

Reply:
moved to the `admin` directory. Still need to replace `@heroicons` with Font Awesome though.
 https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0